air-quality-ui/system_tests/ui/summary_page/summary_page_table_visuals.spec.ts

```diff
@@ -229,4 +231,51 @@ test.describe('Colour testing', () => {
       await summaryPage.assertGridAttributes('colours', expectedTableColours)
     })
   })
+  test('Verify a pollutant value of 0 is assigned AQI 1 colour', async ({
```

Comment:
Could we make it clearer where the value is actually set to 0?

As a maintainer this looks like you're setting it to 1 (though I assume that's AQI?).

Reply:
Yes that's right, we created the createForecastResponseWithValidTimeAndAQI() methods to have values of just 0 for the pollutants so I was re-using this, I can happily be more specific